medatahealthchecker.php

```diff
@@ -142,3 +142,19 @@ function medatahealthchecker_civicrm_entityTypes(&$entityTypes) {
 function medatahealthchecker_civicrm_themes(&$themes) {
   _medatahealthchecker_civix_civicrm_themes($themes);
 }
+
+/**
+ * Implements hook_civicrm_navigationMenu().
+ */
+function medatahealthchecker_civicrm_navigationMenu(&$menu) {
+  $issuesDashboardMenuItem = [
+    'name' => 'medatahealthchecker_issues_dashboard',
+    'label' => ts('Data Health Checker Dashboard'),
+    'url' => 'civicrm/medatahealthchecker/issues-dashboard',
+    'permission' => 'administer CiviCRM,administer MembershipExtras',
```

Comment:
Can you double check the permission `administer MembershipExtras`?
I cannot see it mentioned here or in `uk.co.compucorp.membershipextras` extension's docs/readme

Reply:
> Can you add a workflow action to do the lint checks?

will do

> Can you show more love for phpDoc? 🤞

Unless you think certain classes and/or methods are not clear and could benefit from it and that you can point them out, I don't think there is a need. 

> I don't see a need for the Log suffix in the name of IssuesLog entity, maybe going with Issue is a good one.

Don't know but I feel like `Issue` alone would be a weird name, do you have other suggestions ? if not I will just keep it as is.

> Can you double check the permission administer MembershipExtras?
I cannot see it mentioned here or in uk.co.compucorp.membershipextras extension's docs/readme

It is defined here: https://github.com/compucorp/uk.co.compucorp.membershipextras/blob/master/membershipextras.php#L477

Reply:
> Unless you think certain classes and/or methods are not clear
I can see that simply reading the method reveals what it is supposed to do and that is enough.

> Don't know but I feel like Issue
I see.

Proceed with the workflow action .